### ISSUE TITLE
training loss of quick thoughts

### ISSUE THREAD

**Issue by Zminghua** [BeatHubmann/18F-NLU#2]
hi, I also implemented the quick thoughts method according to the paper “An efficient framework for learning sentence representations”
and i compared my code with yours, I think all is fine.
but the loss has remained unchanged during training.
I check the output probability and find the distribution is uniform.
Have you ever encountered this problem？
thanks

**Comment by BeatHubmann** [comment]
Hi, no that was not one of the problems I had with my code. Obviously this is difficult to diagnose without actually looking at your implementation - have you checked the all weight inits to be non-zero (Xavier etc.)?
Good luck with the debugging!
